docs/pages/Configuration.md

```diff
@@ -536,6 +536,28 @@ You can include certain dependencies into the build process using the `includeDe
 
 This configuration can be overwritten more precisely with the CLI parameters `--include-dependency`, `--include-dependency-regexp`, `--include-dependency-tree`, `--exclude-dependency`, `--exclude-dependency-regexp` and `--exclude-dependency-tree`.
 
+### Minification
+
+For projects of type `application` and `library` the minification is done for all JavaScript files. While the minification step debug-variants are created, original resources are minified and source maps are created. You can exclude the resources of a project from the minification step using a list of glob patterns. Matching resources won't be minified, no debug variants and no source maps are created.
+
+!!! info
+    This configuration is available since UI5 CLI [`v2.14.0`](https://github.com/SAP/ui5-cli/releases/tag/v2.14.0)
+    and applies only to projects defining [Specification Version](#specification-versions)
+    2.6 or higher.
+
```

Comment:
The project's `yaml` file can contain a list of modules declared as glob patterns (resource name patterns) that are excluded from resource minification. Re-includes have to be marked with a leading exclamation mark `!`; see the example below. The order of filters is relevant; a later inclusion overrides an earlier exclusion, and vice versa.

Reply:
done